Makefile

```diff
@@ -103,8 +109,11 @@ test-integration: ## Run integration tests
 .PHONY: test-e2e
 test-e2e: ## Run e2e tests
 	PULL_POLICY=IfNotPresent $(MAKE) docker-build
-	MANAGER_IMAGE=$(CONTROLLER_IMG)-$(ARCH):$(TAG) \
-	go test ./test/e2e -v -tags=e2e -ginkgo.v -ginkgo.trace -count=1 -timeout=90m
+	go test ./test/e2eclusterctl -v -tags=e2eclusterctl -ginkgo.v -ginkgo.trace -count=1 -timeout=90m \
```

Comment:
need to set MANAGER_IMAGE so the kind cluster loads the correct image

Reply:
With the new e2e framework, the MANGER_IMAGE is now defined as part of the E2E config - https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/638/files#diff-61ca65273bbe1a18ee0d5f135d8ab6e7R3-R5

Reply:
the error I observed was that the docker build image is different than the image its trying to load, so the load fails.

Reply:
`./script/ci-e2e.sh` explicitly set the `REGISTRY` environment variable to `e2e`, and I hardcoded image URL in the E2E config to `e2e/cluster-api-azure-controller-amd64:dev`. I could set the hardcoded image URL to `${MANAGER_IMAGE}` and use envsubst to dynamically substitute the value.